tutor/src/components/related-content.cjsx

```diff
@@ -0,0 +1,40 @@
+React    = require 'react'
+_ = require 'underscore'
+
+{ChapterSectionMixin} = require 'shared'
+
+{StepTitleStore} = require '../flux/step-title'
+
+RelatedContent = React.createClass
+
+  propTypes:
+    contentId: React.PropTypes.string.isRequired
+    title: React.PropTypes.string.isRequired
+    chapter_section: React.PropTypes.array.isRequired
+
```

Comment:
hm, line 21 below indicates that `chapter_section` might be a string. In that case perhaps the validation should be:
```
  React.PropTypes.oneOfType([
    React.PropTypes.array React.PropTypes.string
  ]).isRequired
``